### ISSUE TITLE
[Neat URL] Breaks Pocket / Firefox Accounts OAuth

### ISSUE THREAD

**Issue by callahad** [Smile4ever/firefoxaddons#86]
Using Firefox Nightly and Neat URL 3.2.0.

STR:

1. (If applicable) Sign out of getpocket.com
2. Navigate to a page and click the Pocket icon in the URL bar
3. Click "Sign in with Firefox"

What should happen:

- I see the Firefox Accounts sign in screen

What actually happens:

- I see an error about invalid OAuth parameters

Why this happens:

Neat URL appears to encode spaces as `+`, rather than a literal space or `%20`. For example:

```
Unmodified: https://accounts.firefox.com/oauth/signin?client_id=xxx&state=xxx&scope=profile:uid profile:email profile:display_name
w/Neat URL: https://accounts.firefox.com/oauth/signin?client_id=xxx&state=xxx&scope=profile:uid+profile:email+profile:display_name 
```

Note specifically `scope=profile:uid profile:email profile:display_name` vs `scope=profile:uid+profile:email+profile:display_name`

**Comment by Smile4ever** [comment]
Have you tried the latest Neat URL git code?

**Comment by callahad** [comment]
Ah, yep, this was resolved by 0e5a92662ae3e159740337ea23adbec0cac97c42
